index.js

```diff
@@ -0,0 +1,52 @@
+/*eslint-disable */
```

Comment:
- [ ] Kindly fix these Linters issues instead of disabling them. Linters are great tools that flag errors in your code. Let's fix these issues in all the files you have disabled Linters.